OsmAnd/res/values/strings.xml

```diff
@@ -12,21 +12,21 @@
 -->
 
     <string name="thank_you_for_feedback">Thank you for feedback</string>
-    <string name="poi_cannot_be_found">Node or way cannot be found.</string>
-    <string name="search_no_results_feedback">No search results?\nGive us feedback</string>
+    <string name="poi_cannot_be_found">Could not find node or waypoint.</string>
+    <string name="search_no_results_feedback">No search results?\nProvide feedback</string>
     <string name="release_3_1">
     • Navigation: Fix progress bar, fast swapping of the start and end point of the route\n\n
-    • Map markers: fix turn on/off groups, ability to hide markers from the map\n\n
+    • Map markers: Fix turn on/off groups, ability to hide markers from the map\n\n
```

Comment:
Why include an initial uppercase letter on the first word after a colon? This is not common style.

Reply:
Both are allowed in English, whereas in other languages smaller versal following colon is an active error. In effect doing it this way produces fewer errors in translation.

Reply:
From what I've read, an initial uppercase letter after a colon is generally only used occasionally. I think it's poor style, because it removes the ability to indicate proper nouns.

I don't think it's acceptable to change English strings to worse ones to ensure better translation. If you're concerned at translators taking the English strings too literally, perhaps more guidance could be given, but why should the English strings suffer?

Reply:
Proper nouns are still in proper form that way, with the possibility of being misunderstood, for a somewhat slippery slope of allowing both forms in a language, or clear error where adherence to consistency in using only colon bigversal is maintained.(Like with OsmAnd at large, this change now reflecting that.)
In any event care could be taken when making a proper noun, or incorporating a badly made one in a sentence. I would argue big versal following colon isn't much of an issue in English stylistically, whereas it really is in other languages.
The sentence reads "Fix turn on/off groups", it should imo. read "turning", and it is a bigger fish to fry.

Reply:
With your style, proper nouns are in proper form, but only as a consequence of the blind rule that the first letter of a word following a colon should be in uppercase. How can you tell that the word is a proper noun?

If you don't use an initial uppercase letter for the first word following a colon, you're:

* following the most common style in English
* preserving the ability to indicate a proper noun

As such, I see little argument for using an initial uppercase letter for the first word after a colon. Leave the translations to experts in other languages. If someone makes a mistake there, others can always suggest an improvement. Please don't make the English strings worse out of a well-intentioned idea for helping translators. I'd argue that the best translations will come from looking at the source code and application itself to see how the strings are used.

Reply:
You could put it in italics, quote it, or just write your way around ( on the assumption that neither is correct. Never had the problem to begin with.

The "this is not the most common style in English", is not a major upset, whereas seeing a small versal following a colon in other indo-european languages is. Thankfully it is consistent, so this error can be avoided altogether by just opting for colon bigversal.

Translations in this case are just equally good, in being correct, as opposed to being, as a causal effect, fully wrong. The libre software translators are not experts, nor am I. The result is invariably, always the same. It is a trap that works.

Yes, it could change in time, but doesn't. This level of consistency is predicated on innate consideration. Give it enough time, and that makes for an academic argument. Meanwhile, it is a small concession to make, which increases the perceived quality of the product by a lot in languages other than English. At no point in time should anyone be exposed to it.

Also, it is how every other string is styled already. If you think about it, I am that person that comes along and ensures consistency. This was the case before I made any changes too. Now nobody has to see errors, because this hasn't landed yet.
In any event, doing both in one corpus is clearly worse than either.